src/cryptography/x509/extensions.py

```diff
@@ -1483,6 +1480,16 @@ def __ne__(self, other):
         return not self == other
 
 
+@utils.register_interface(ExtensionType)
+class PrecertificateSignedCertificateTimestamps(SCTBase):
```

Comment:
I realize these are only being used to change the name of the class and the OID associated, but despite that I'm not wild about having a subclass. We have been very successful at not subclassing anything in this project. @alex do you have an opinion?

Reply:
I agree with you.

Reply:
@alex so what do we want to do here. Duplicate this class and its attendant tests?

Reply:
That's roughly what we do for all of the "this is an iterable" extensions, yes. If we want to think about reducing that duplication, there are things we could consider.